Tests/SwiftSDKGeneratorTests/EndToEndTests.swift

```diff
@@ -319,17 +333,28 @@ func buildTestcases(config: SDKConfiguration) async throws {
       logger, scratchPath: tempDir.path, withArguments: config.sdkGeneratorArguments)
   }
 
-  logger.info("Built SDK")
+  logger.info("Built Swift SDK")
+
+  // Cleanup
+  let cleanupSdk: () async -> Void = {
```

Comment:
We tend to preserve uppercased abbreviation letters with Swift API naming convention, i.e. `URL`, not `Url`. Also, `func` feels more readable in this context to me

```suggestion
  func cleanupSDK() async {
```